watertap/core/zero_order_costing.py

```diff
@@ -2022,9 +2195,9 @@ def _get_tech_parameters(blk, parameter_dict, subtype, param_list):
                 pblock.add_component(p, vobj)
             except KeyError:
                 raise KeyError(
-                    "Error when trying to retrieve costing parameter"
-                    " from {p} database. Please check the YAML "
-                    "file for this technology for errors."
+                    "Error when trying to retrieve costing parameter "
+                    "for {p}. Please check the YAML "
+                    "file for this technology for errors.".format(p=p)
```

Comment:
The way this was written, it was intended to be an `f-string` which would be cleaner and maybe slightly faster.